tests/core/osac_cli.py

```
@@ -73,6 +74,13 @@ def create_compute_instance(
             run_strategy,
         ]
 
+        if instance_type is not None:
+            if cores is not None or memory_gib is not None:
+                raise ValueError("Cannot specify cores/memory_gib together with instance_type")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)